pelita/game.py

```diff
@@ -669,28 +675,30 @@ def apply_move(gamestate, bot_position):
             food[1 - team].remove(bot_position)
             # This is modifying the old game state
             score[team] = score[team] + 1
-    # check if anyone was eaten
+    # check if we killed someone
     if bot_in_homezone:
         killed_enemies = [idx for idx in enemy_idx if bot_position == bots[idx]]
         for enemy_idx in killed_enemies:
             _logger.info(f"Bot {turn} eats enemy bot {enemy_idx} at {bot_position}.")
```

Comment:
I think we should put this (the scoring and resetting part) into its own function at some point do avoid errors coming from the duplication.

Reply:
For after Camerino